chart/flux/templates/deployment.yaml

```diff
@@ -220,7 +220,7 @@ spec:
           {{- if .Values.manifestGeneration }}
           - --manifest-generation=true
           {{- end }}
-          - --registry-poll-interval={{ .Values.registry.pollInterval }}
+          - --automation-interval={{ .Values.registry.pollInterval }}
```

Comment:
Would you be able to change the `pollInterval` to `automationInterval` too, in a backwards compatible manner?

Reply:
Like this?